_includes/docs/user-guide/rpc.md

```diff
@@ -82,6 +82,36 @@ This token should belong to either
  
 You can use the following [guide](/docs/{{docsPrefix}}reference/rest-api/#rest-api-auth) to get the token.
 
+## Persistent RPC
+
+ThingsBoard provides a new and unique feature: **Persistent RPC**. 
```

Comment:
we can't guarantee that the feature is unique and other platforms don't have it.

Reply:
Please indicate "since version ..."